dftimewolf/lib/processors/turbinia_gcp.py

```diff
@@ -149,26 +113,59 @@ def _DownloadFilesFromGCS(self, timeline_label, gs_paths):
         # Don't add a critical error for now, until we start raising errors
         # instead of returning manually each
         self.ModuleError(str(exception), critical=False)
-      self.logger.info('Downlaoded {0:s} to {1:s}'.format(path, local_path))
+      self.logger.info('Downloaded {0:s} to {1:s}'.format(path, local_path))
 
       if local_path:
         local_paths.append((timeline_label, local_path))
 
     return local_paths
 
-  def Process(self):
-    """Process files with Turbinia."""
-    log_file_path = os.path.join(self._output_path, 'turbinia.log')
-    self.logger.info('Turbinia log file: {0:s}'.format(log_file_path))
-    vm_containers = self.state.GetContainers(containers.ForensicsVM)
-    if vm_containers and not self.disk_name:
-      forensics_vm = vm_containers[0]
-      self.disk_name = forensics_vm.evidence_disk.name
-      self.logger.info(
-          'Using disk {0:s} from previous collector'.format(self.disk_name))
+  def TurbiniaSetUp(self, project, turbinia_zone, sketch_id, run_all_jobs):
+    """Sets up the object attributes.
+    Args:
+      project (str): name of the GPC project containing the disk to process.
```

Comment:
s/GPC/GCP/.   Also, super minor nit, but you probably want a newline above the args section.